src/ast/mod.rs

```diff
@@ -857,6 +857,8 @@ pub enum Statement {
         default_charset: Option<String>,
         collation: Option<String>,
         on_commit: Option<OnCommit>,
+        clickhouse_order: Option<Vec<Ident>>,
+        query_after: Option<Box<Query>>,
```

Comment:
Can you please add a comment here that this supports a query at the end of the statement (aka for clickhouse)